src/Symfony/Component/Form/Extension/Core/DataTransformer/PercentToLocalizedStringTransformer.php

```diff
@@ -116,6 +116,7 @@ public function reverseTransform($value)
             return;
         }
 
+        $position = 0;
```

Comment:
shouldn't we remove this variable? It's never ever changed, so inlining `0` should be just fine.

Reply:
OK, nevermind :)